lib/mayu/vdom/update_context.rb

```diff
@@ -35,7 +35,7 @@ def dom_parent_id = @dom_parent_ids.last || "probably root"
       def enter(vnode, &blk)
         # Sleep so that the fiber yields,
         # so that other things can run..
-        sleep(0)
+        # sleep(0)
```

Comment:
Did this even do anything?

Reply:
Seems like it does. It seems to yield the fiber which makes things a little bit slower but other work can be done at the same time.